imap_processing/codice/constants.py

```diff
@@ -76,73 +79,85 @@
 DATA_PRODUCT_CONFIGURATIONS = {
     CODICEAPID.COD_HI_INST_COUNTS_AGGREGATED: {
         "num_counters": 1,
-        "num_energy_steps": 128,
+        "num_energy_steps": 1,
+        "num_spin_sectors": 6,
```

Comment:
Are all of the data products going to have these same dimensions? I would have assumed each product would have the same "spin_sectors" (or at least consistent between hi/lo)

Reply:
Thank you, your comment made me realize that I messed up the logic a bit here. What I am calling "spin sectors" here is actually a representation of "number of positions with 12 spin angles", and so I need to restructure the data a bit differently. Will address this in an upcoming commit.